openstates/settings.py

```diff
@@ -7,7 +7,11 @@
 SCRAPELIB_TIMEOUT = 60
 SCRAPELIB_RETRY_ATTEMPTS = 3
 SCRAPELIB_RETRY_WAIT_SECONDS = 10
-SCRAPELIB_VERIFY = True
+try:
+    verify = bool(os.environ.get("VERIFY_CERTS", False))
```

Comment:
Will this actually do what you mean for it to? If I remember correctly bool("FALSE") is still True, because all non-blank strings are True.

Reply:
Yeah...good point. Here's me forgetting how type casting works in python.

Reply:
I suppose would could just not allow for environment-level configuration?

Reply:
:+1: that bool thing has gotten me so many times...